subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -230,6 +230,21 @@ This enables watching the file system for this build only.
 Put `org.gradle.unsafe.watch-fs=true` in your `gradle.properties`::
 This enables watching the file system for all builds, unless explicitly disabled with `--no-watch-fs`.
 
+Here are some operating system specific notes.
+
+===  Linux
+
+On Linux, file system watching uses inotify.
+Depending on the size of your build, it may be necessary to increase inotify limits.
+
+[[sec:inotify_watches_limit]]
+Inotify watches limit::
+To raise the inotify watches limit see https://confluence.jetbrains.com/display/IDEADEV/Inotify+Watches+Limit[here].
+
+[[sec:inotify_instances_limit]]
+Inotify instances limit::
+To raise the inotify instances limit see https://confluence.jetbrains.com/display/IDEADEV/Inotify+Instances+Limit[here].
+
```

Comment:
Why are we linking to JetBrains docs? I think we should instead copy that part, or link to inotify's own docs or something.